src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.CompilationTracker.cs

```diff
@@ -191,145 +191,71 @@ public ICompilationTracker FreezePartialState(CancellationToken cancellationToke
             {
                 var state = this.ReadState();
 
-                // If we're already finalized then just return what we have, and with the frozen bit flipped so that
```

Comment:
all this work got effectively inlined into this method.

Reply:
the diff is *awful*.  i recommend not looking at it inlined.

Reply:
ok.  because the diff was so awful, i moved this method much lower.  so it just shows as a full add.